representation/transparency_using_selection/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { createRootViewer } from "./common/init";
+import { setStructureTransparency } from "molstar/lib/mol-plugin-state/helpers/structure-transparency";
+import { Queries, QueryContext, StructureProperties, StructureSelection } from "molstar/lib/mol-model/structure";
+
+async function init() {
+    // Create viewer
+    const plugin = await createRootViewer();
+    
+    // Download PDB
+    const fileData = await plugin.builders.data.download(
+        { url: "https://models.rcsb.org/4hhb.bcif", isBinary: true }
+    );
+
+    // Load PDB and create representation
+    const trajectory = await plugin.builders.structure.parseTrajectory(fileData, "mmcif");
+    const data = await plugin.builders.structure.hierarchy.applyPreset(trajectory, "default");
+
+    // Get Structure and StructureRef objects from data
+    const struct = data?.structure.cell?.obj?.data!;
```

Comment:
To avoid the null check on the `data` collection, we can add an assertion before:
```
if (!data) {
        throw new Error("Structure not loaded");
    }
```

also, `structure.cell.obj` points to the same reference as `structure.obj`
```
const struct = data.structure.obj!.data!;
```
The TS `!` assertion is better in this case because then struct can not be undefined from TS POV

Reply:
We can even use `data` property getter from the structure which points to obj.data
```ts
const struct = stateObjects.structure.data
```